dts/bindings/net/wireless/silabs,series2-radio.yaml

```diff
@@ -56,3 +56,12 @@ properties:
       property is not set, no sub-GHz PA is enabled.
     enum:
       - highest
+
+  ble-tx-high-power-supported:
```

Comment:
Even though this isn't based on any spec, but since there's a property with the identical name in the nRF binding, perhaps we'd still want this in the common base binding to avoid duplication (with a mention in the description that the meaning is implementation-specific?). Thoughts from @cvinayak?

Reply:
see https://github.com/zephyrproject-rtos/zephyr/pull/97994/files#r2451718718

`tx-high-power-supported` is very generic to wireless.  Can we or do we already have a `generic,radio.yaml` ?  this should go there.